indico/web/assets/vars_js.py

```diff
@@ -117,6 +117,10 @@ def generate_global_file(config):
             },
 
             'Timetable': {
+                'entry': {
+                    'move': url_rule_to_js('timetable.move_timetable_entry'),
+                    'changeDates': url_rule_to_js('timetable.change_start_date')
```

Comment:
That is a side effect that will be taken care of by what we call `auto-ops` and I think it should be reflected in the endpoint. I prefer datetime rather than time, because the date will change for sure.